client/landing/gutenboarding/stores/domain-suggestions/index.ts

```diff
@@ -6,5 +6,5 @@ import { getSuggestionsVendor } from 'lib/domains/suggestions';
 
 export const DOMAIN_SUGGESTIONS_STORE = DomainSuggestions.register( {
 	/* Returns an ID for the domain suggestions vendor. Passing `true` to getSuggestionsVendor returns the signup variant.*/
-	vendor: getSuggestionsVendor( true ),
+	vendor: getSuggestionsVendor( { isSignup: true } ),
```

Comment:
This has been implemented in https://github.com/Automattic/wp-calypso/pull/44869/files#diff-7a5e069c8aa791358596a1fa6df0cf6b9aca820c66bab285287441b2ce752c98

Reply:
Thanks for flagging this, @razvanpapadopol! I am going to close this PR.